CONTRIBUTING.rst

```diff
@@ -46,6 +67,12 @@ that `pre-commit` may change some of the files to be committed. When this happen
 by re-staging the files, then committing again. This gives you a chance to review the changes brought about by
 `pre-commit`.
 
+Testing Policy
+--------------
+
+The RefRed project make use of automated testing.
+Pull requests should bear this in mind and note that requests that increase the test coverage are heavily preferred.
+Requests will be automatically annotated with code coverage results to aid the contributor and development team.
 
```

Comment:
I didn't see in this file a comment about how to add and use data files in the context of testing. Also a word to encourage the developer to work with the data (especially the event files) that already exists in the data directory, instead of adding more files